client/src/api/recipe.js

```diff
@@ -15,6 +15,9 @@ const urls = {
   updateIsPublished(id) {
     return `${this.root}/update-published/${id}`;
   },
+  showRecipe(id) {
+    return `${this.root}/show-recipe/${id}`;
```

Comment:
if you change the backend and remove the `show-recipe` don't forget to change this